index.html

```diff
@@ -6,13 +6,16 @@
     <title>Document</title>
 </head>
 <body>
-    <head>
+    <header>
         <h1>Web Desarrollo</h1>
         <nav>
             <a href="#"></a>
             <a href="#"></a>
             <a href="#"></a>
         </nav>
-    </head>
+    </header>
+    <aside>
```

Comment:
Añade un h2 para poner un subtitulo antes del <p>.
```suggestion
    <h2></h2>
```